MainDemo.Wpf/TransitionsDemo/Slide5_TransitioningContent.xaml

```diff
@@ -36,7 +36,7 @@
                     <Setter Property="Mode" Value="PrimaryLight" />
                 </Style>
             </ItemsControl.Resources>
-            <materialDesign:TransitioningContent OpeningEffect="{materialDesign:TransitionEffect ExpandIn}">
+            <materialDesign:TransitioningContent OpeningEffect="{materialDesign:TransitionEffect Kind=ExpandIn}">
```

Comment:
OK, cool for the constructor to get rid of VS errors, but can we drop the "Kind="?  Just means then users see the must succinct syntax